extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -43,7 +43,8 @@ class AmpAccordion extends AMP.BaseElement {
       log.warn(TAG, `Experiment ${EXPERIMENT} disabled`);
       return;
     }
-    this.sections_.forEach(section => {
+    this.element.setAttribute('role', 'tablist');
```

Comment:
You will also have to set the initial value on whichever panels have `enabled` set by the publisher.